components/script/dom/element.rs

```diff
@@ -205,6 +205,7 @@ pub struct Element {
     #[no_trace]
     selector_flags: Cell<ElementSelectorFlags>,
     rare_data: DomRefCell<Option<Box<ElementRareData>>>,
+    is_element_overflow_value_propagated: Cell<bool>,
```

Comment:
Seems weird to store this information, which is purely about layout, in this struct.
But if you must store it here, at least move it to the rare data.